src/runtime/web/remote.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+import { fetch } from './io';
+import { Remote } from '../../bindings/html/remote';
+
+export class WebRemote extends Remote {
+  constructor(requestedLangs) {
+    super(requestedLangs);
```

Comment:
I move the `Remote` back into the bindings, and I'd like to extend it in the runtime.  A `remote` needs two runtime-specific things:  a `fetch` method and a `broadcast` method.  The implementation depends on the client-service architecture.

I'd like to extend the base `Remote` here, but Gecko doesn't support `super` at all (https://bugzil.la/1066239).  @zbraniecki any suggestions how to handle this?  I could re-write `Remote` to use the es5 `prototype` way of defining constructors and methods and then `Remote.call(this, requestedLangs)` here.  What do you think?